core/classes/Events/EventHandler.php

```diff
@@ -28,41 +28,27 @@ public static function registerWebhooks(array $webhooks): void
      * Register an event.
      * This must be called in the module's constructor.
      *
-     * @param class-string|string $event       Name of event to add.
-     * @param string              $description Human readable description.
-     * @param array               $params      Array of available parameters and their descriptions.
-     * @param bool                $return      Whether to return $params afterwards
-     * @param bool                $internal    Whether to hide this hook from users in the StaffCP (ie for internal events)
+     * @param  class-string $event Event to add.
```

Comment:
```suggestion
     * @param  class-string<AbstractClass> $event Event to add.
```

This should work nicely here